build-image/README.md

```diff
@@ -50,3 +50,20 @@ to the root contract:
 It is a known issue that running AMD64 image on the ARM host architecture significantly impacts the performance
 and build times. To solve this issues, enable _Use Rosetta for x86/amd64 emulation on Apple Silicon_ in
 _Settings_ -> _Features in development_ tab in Docker Desktop App.
+
+**How to debug**
+
+Build docker image:
+
+```bash
+docker buildx build -t paritytech/contracts-verifiable:0.0.1 .
```

Comment:
That's a strange tag. Where does `0.0.1` come from?

Reply:
I've set a tag to differentiate the locally built image from the one downloaded. It can be any tag, do you want to change it?